### PR TITLE
Add maisi bundle

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: ci/run_premerge_cpu.sh [hunk 1]
```diff
@@ -52,21 +52,26 @@ verify_bundle() {
             echo $bundle_list
             for bundle in $bundle_list;
             do
-                pip install -r requirements-dev.txt
-                # get required libraries according to the bundle's metadata file
-                requirements=$(python $(pwd)/ci/get_bundle_requirements.py --b "$bundle")
-                # check if ALLOW_MONAI_RC is set to 1, if so, append --pre to the pip install command
-                if [ $ALLOW_MONAI_RC = true ]; then
-                    include_pre_release="--pre"
+                if [ "$bundle" != "maisi_ct_generative" ]; then
```
Comment (KumoLiu) [inline ci/run_premerge_cpu.sh]: We didn't have a formal way to skip verify bundle right?

File: ci/run_premerge_gpu.sh [hunk 2]
```diff
@@ -16,95 +16,123 @@
...
-
-    verify_bundle)
-        verify_bundle
-        ;;
-
-    *)
-        echo "ERROR: unknown parameter: $BUILD_TYPE"
-        ;;
-esac
+verify_bundle
```
Comment (KumoLiu) [inline ci/run_premerge_gpu.sh:137]: Does this change mean will always verify all the bundle on the GPU?

Comment (yiheng-wang-nv) [reply ci/run_premerge_gpu.sh:137]: yes, but the old logic is the same. The main purpose is to remove unnecessary logic

File: ci/run_premerge_cpu.sh (hunk 1 above)
Comment (yiheng-wang-nv) [reply ci/run_premerge_cpu.sh]: added a list to contain it

**Review by KumoLiu — APPROVED** [review]

**Pull request merged**